variables.tf

```diff
@@ -404,3 +404,9 @@ variable "monitor_reader_principal_id" {
   description = "Principal(Object) ID of a user/group which would receive notifications from alerts."
   type        = string
 }
+
+variable "scaleset_actions_recipients_email_list" {
```

Comment:
I would make this more abstract so we can reuse it for other email notifications if needed.

Reply:
Done